packages/bff/package.json

```diff
@@ -37,11 +37,12 @@
     "axios": "1.7.2",
     "dotenv-flow": "4.1.0",
     "express": "4.19.2",
-    "pagopa-interop-commons": "workspace:*",
+    "jszip": "3.10.1",
```

Comment:
Since its latest release is 2 years old, how about using [adm-zip](https://github.com/cthackers/adm-zip) which is actively developed?

Reply:
@AsterITA this [commit](https://github.com/pagopa/interop-be-monorepo/pull/882/commits/e8ad79fac1d843e91376d0de72e8abc9f1b73499) add adm-zip lib.